server/rpm/pbench-server.spec.j2

```diff
@@ -8,30 +8,11 @@ URL:            http://github.com/distributed-systems-analysis/pbench
 Source0:        pbench-server-%{version}.tar.gz
 Buildarch:      noarch
 
-
-%if 0%{?rhel} == 7
```

Comment:
At this point, I don't disagree and I'm not going to complain, especially given the pressure you were under last night doing this ... I know at one point when we'd discussed the RHEL 7 addition of "native" Python we'd made a deliberate decision to leave 0.69 alone. I'm OK with changing that, but just pointing it out for posterity ... ;-0

Reply:
Sure.  I did not want to have to verify this still works for RHEL 7.

Basically, these changes are declaring that we no long support RHEL 7 for the `b0.69` branch.